apps/api/src/server.ts

```diff
@@ -20,16 +20,42 @@
 import { demoApplication } from '@pcgl-daco/data-model/src/main.js';
 import cors from 'cors';
 import express, { Request, Response } from 'express';
+import path, { dirname } from 'path';
+import * as swaggerUi from 'swagger-ui-express';
+import { fileURLToPath } from 'url';
+import yaml from 'yamljs';
+
+import { getHealth, Status } from './app-health.js';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
 
 export const port = process.env.PORT || 3000;
 
 const startServer = async () => {
 	const app = express();
 
+	app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(yaml.load(path.join(__dirname, './resources/swagger.yaml'))));
+
 	app.get('/', (_req: Request, res: Response) => {
 		res.send('Hello World!');
 	});
 
+	app.get('/health', (_req: Request, res: Response) => {
+		const health = getHealth();
+
+		// TODO: implement proper versioning logic when in place
+		const resBody = {
+			version: `PCGL-V1`,
+			health,
+		};
+		if (health.all.status == Status.OK) {
+			res.send(resBody);
+			return;
+		}
+		res.send(resBody);
```

Comment:
What is the benefit of the `if` statement? In both cases `res.send(resBody)` gets called then the function exits

Reply:
Woops nice catch, I was debugging the response object and removed the last functionality and forgot to add it back in: `res.status(500).send(resBody)` and `res.status(200).send(resBody)` is what it was before.